src/core/packages/chrome/layout/core-chrome-layout/layouts/grid/grid_global_app_style.tsx

```diff
@@ -28,7 +29,10 @@ const globalLayoutStyles = (euiTheme: UseEuiTheme['euiTheme']) => css`
     --kbnHeaderBannerHeight: var(--kbn-layout--banner-height, 0px);
```

Comment:
nit: I would like these to follow the same style we adopted for the grid layout variables, e.g. --kbn-header--banner-height (or whatever suits).